packages/mobile/src/account/__snapshots__/Account.test.tsx.snap

```diff
@@ -14,8 +14,10 @@ exports[`Account renders correctly 1`] = `
       Object {
         "alignItems": "center",
         "backgroundColor": "transparent",
-        "borderBottomColor": "#EDEEEF",
-        "borderBottomWidth": 0,
+        "borderBottomColor": AnimatedValue {
+          " __value": NaN,
```

Comment:
I was not able to have `0` here, it's only a problem in snapshots. Lmk if anyone know how to fix `interpolateColors` with jest.